src/libAtomVM/ets.c

```diff
@@ -388,3 +420,117 @@ EtsErrorCode ets_delete(term ref, term key, term *ret, Context *ctx)
 
     return EtsOk;
 }
+
+bool operation_to_tuple4(term operation, term *position, term *increment, term *threshold, term *set_value)
```

Comment:
This is a private function used only in this source file, lets declare it as `static`:
`static bool operation_to_tuple4(...)`